client/transport/tlsconfig_clone.go

```diff
@@ -1,4 +1,4 @@
-// +build !go1.7,!windows
```

Comment:
I'm not sure why there was `!windows` here, Clone() seems supporting windows.
https://github.com/golang/go/commit/d24f446a90ea94b87591bf16228d7d871fec3d92